pkg/cmd/run.go

```diff
@@ -106,7 +106,7 @@ func performPrerun() error {
 		return err
 	}
 
-	return prerun.New(dataManager).Perform()
+	return management.New(dataManager).Perform()
```

Comment:
We might want a new method name here. "management perform" isn't very specific about what's going to happen.

Reply:
Yeah, agreed. Thinking more along the lines of `management.NewPreRun().Validate()`

Reply:
I was more thinking about keeping `prerun` package, but rename the `PreRun` struct to e.g. `BackupManagement` since this is what it does - backs up, restores, or deletes the data, so maybe even `DataManagement`?

Reply:
Sounds good, based off of our slack convo, went ahead and changed it to `BackupManager`, that does make more sense since it's doing all the backup management tasks.